service/bblfsh/bblfsh.go

```diff
@@ -121,6 +111,7 @@ func (s *BaseScanner) parseFile(f *lookout.File) (*uast.Node, error) {
 		Filename: f.Path,
 		Content:  string(f.Content),
 		Encoding: protocol.UTF8,
+		Language: strings.ToLower(f.Language),
```

Comment:
why do we want to convert to lowercase here?

Reply:
because otherwise bblfsh returns error like `driver for language Go isn't installed`.